AnastasiiaKaravaieva/HW_7/3.py

```diff
@@ -0,0 +1,9 @@
+def calculating_character(word):
```

Comment:
please look at this implementation

def character_count(word: str) -> dict:
    """
    The function returns a dictionary, where keys are characters from
    entered string and values are numbers of their appearances.
    """
    char_count = {}
    for char in word:
        if char not in char_count:
            char_count[char] = word.count(char)
    return char_count